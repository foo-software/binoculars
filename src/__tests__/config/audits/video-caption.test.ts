import auditVideoCaption from '../../../config/audits/video-caption';

describe('video-caption audit', () => {
  it('should match snapshot', () => {
    expect(auditVideoCaption('en')).toMatchSnapshot();
  });
});
