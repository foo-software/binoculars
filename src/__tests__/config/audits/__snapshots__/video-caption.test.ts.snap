// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`video-caption audit should match snapshot 1`] = `
Object {
  "description": "When a video provides a caption it is easier for deaf and hearing impaired users to access its information. [Learn more](https://web.dev/video-caption/).",
  "title": "\`<video>\` elements contain a \`<track>\` element with \`[kind=\\"captions\\"]\`",
}
`;
